dev/bench/bench.sh

```diff
@@ -70,15 +70,15 @@ check_variable () {
 : "${old_ocaml_flambda:=0}"
 : "${new_coq_repository:=$CI_REPOSITORY_URL}"
 : "${old_coq_repository:=$CI_REPOSITORY_URL}"
-: "${new_coq_opam_archive_git_uri:=https://github.com/coq/opam-coq-archive.git}"
+: "${new_coq_opam_archive_git_uri:=https://github.com/mattam82/opam-coq-archive.git}"
```

Comment:
no such repo found, is it private?

Reply:
No, it's the usual opam-coq-archive vs coq-opam-archive confusion :)